http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1927,16 +1880,40 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
-  /proxy/invalidateToken:
+  "/proxy/setCookie":
+    get:
+      summary: Sets a browser cookie needed to authorize connections to a Leonardo runtime
```

Comment:
Am I understanding the change correctly - that this endpoint is used to auth any runtime rather than just specific instance? Is there an implementation change associated with this - or was the project/namespace purely excess info to begin with?

Reply:
nvm - it's called reading the PR description 🤦‍♂️